README.rst

```diff
@@ -184,6 +184,8 @@ Usage
    Note: If after closing a program with graphical interface (i.e. FSLeyes)
    LXterminal does not raise the shell ($) prompt then press Ctrl + C
    to finish closing the program.
+   
+   Note2 : If you are having connectivity problems (errors related to connections), try restarting your computer. If the installation is done in one go (without ever rebooting) it is likely you will end up with connectivity problems.
```

Comment:
@Nicolas-Pinon i'm not sure i fully understand the sentence (also, not sure if you are talking about issue during installation or during use). I'm trying to rephrase, let me know if it's accurate:

Note2: If you experience issues with Internet connection during installation, reboot the computer and re-launch the installation procedure.

Reply:
It was during the installation, I did not need to re-launch all the installation procedure though.
The thing is, it is not really an "internet" connection problem, it has to do with a port forwarding thing (as we talk about with @zougloub , but I don't understand everything here).
But from what I understand it is an "internal connectivity problem", like the Virtual Machine trying to connect to a port on the real machine.

Let me know if you think this sentence is clearer : 

Note2 : If you are having errors stating that you can't connect to the 2222 port or any connectivity issue, try restarting your computer. If the installation is done in one go (without ever rebooting) it is likely you will end up with connectivity problems.

Thing is I didn't want to be too specific on my former sentence (stating explicitely the port 2222 thingy) because I thing that as a general rule if the installation is not working, rebooting the computer is a good idea.

Reply:
ok i see, how about that:

Note2: If you experience errors such as "cannot connect to port 2222" or any connectivity-related issue, reboot your computer. If you keep having problems, please open an issue [here](https://github.com/neuropoly/sct_docker/issues).

Reply:
ok, in that case i would not put it as a separate note, but as an official step inside "Installation":

7. Reboot your computer

Reply:
Done, is everyone ok with it ?

Reply:
👍

Reply:
From experience... give a reason or people won't reboot their computer, thinking it's useless.

Reply:
Good.